Packet++/header/LineBasedProtocol.h

```diff
@@ -0,0 +1,55 @@
+#ifndef PACKETPP_LINE_BASED_PROTOCOL_LAYER
+#define PACKETPP_LINE_BASED_PROTOCOL_LAYER
+
+#include "Layer.h"
+
+/// @file
+
+/**
+ * \namespace pcpp
+ * \brief The main namespace for the PcapPlusPlus lib
+ */
+namespace pcpp
+{
+
+	/**
+	 * Class for general line based protocol (FTP, SMTP) message
+	 */
+	class LineBasedProtocolMessage : public Layer
```

Comment:
Is there a "formal" name for this format that is used in FTP and SMTP?
I'm not sure "line-based" is very clear...

Reply:
To be honest, I don't know too 😁. They are actually TextBased but differ slightly from HTTP. I will look for a better naming until your second pass.

Reply:
Actually maybe `SingleCommandTextProtocol`?